### DISCUSSION TITLE
[A-L] (3 Aprile 2023) Esercizio 4

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#383] [Esami A-L]
Dimostrare per Induzione che, per ogni $\Large n \geq 0,\ n(n + 1)$ è pari.

Indicare con precisione:

1. Caso Base.

2. L’ipotesi induttiva.

3. Dimostrazione del passo induttivo.

**Comment by FrancescoContinella** [comment]
<img width="1037" height="758" alt="Screenshot 2025-07-22 155448" src="https://github.com/user-attachments/assets/d6da68db-58d5-4677-b3dc-c35aec3cc16f" />
